### DISCUSSION TITLE
GitHub Repository to Azure Web App Deployment- "The value of SourceRoot.RepositoryUrl with identity XX is invalid"

### DISCUSSION THREAD

**Discussion by richarddefreitas** [community/community#132393] [Programming Help]
### Select Topic Area

Question

### Body

Good morning everyone!

I have uploaded a .NET 8.0 application to GitHub and am trying to setup a deployment Action to push to Azure Web Apps (CI/CD, upon Commit). The YAML file keeps failing on the dotnet build and publish step, giving an error of 

/home/runner/.nuget/packages/microsoft.sourcelink.azurerepos.git/8.0.0/build/Microsoft.SourceLink.AzureRepos.Git.targets(28,5): error : The value of SourceRoot.RepositoryUrl with identity '/home/runner/work/APPNAME/APPNAME/' is invalid: 'https://github.com/GITHUBNAME/REPONAME' [/home/runner/work/APPNAME/APPNAME/APPNAME.csproj]
Build FAILED.

I can build and deploy the application using Visual Studio Community 2022, and am not explicitly using SourceLink anywhere in my application. Does anyone have any idea what is causing this or how to eliminate it so I can get a successful build and deploy?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @richarddefreitas ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
